src/Components/Common/UserDetailsComponet.tsx

```diff
@@ -0,0 +1,25 @@
+import UserDetails from "./UserDetails";
+
+const UserDetailComponent = ({
+  id,
+  title,
+  value,
+}: {
+  id: string;
+  title: string;
+  value: string;
+}) => (
+  <div className="col-span-1">
+    <UserDetails id={id} title={title}>
+      <div
+        className="overflow-hidden font-semibold"
+        style={{ textOverflow: "ellipsis" }}
```

Comment:
Use tailwindcss
```suggestion
```

Reply:
> Use tailwindcss

Hi, I've made the requested changes please see.